features/step_definitions/journey/registration_steps.rb

```diff
@@ -0,0 +1,70 @@
+# Group pieces together when they are always consecutive (renewal - new lower tier - new upper tier)
+# Helpers vs Step to group things together - Choose helpers, avoid steps.
+# Helpers vs Step - repeat helpers, not steps.
+# Use `Given` to set up instance variables for the rest of the journey decisions
+
+Given("I want to register as a lower tier carrier") do
+  load_all_apps
+
+  @tier = :lower
+end
+
+When("I start a new registration journey in {string} as a {string}") do |location, organisation_type|
+  @organisation_type = organisation_type
+
+  @journey.start_page.load
+  @journey.start_page.submit
+
+  @journey.location_page.submit(choice: location)
+
+  @journey.business_type_page.submit(org_type: organisation_type)
+end
+
+When("I complete my registration") do
+  if @organisation_type != "charity"
+    @journey.check_your_tier_page.submit(option: :unknown)
```

Comment:
This looks correct, but there's a current bug where this page does not show for overseas registrations. See RUBY-970, issue 1.

Reply:
well, this page is new, and I don't have a case for overseas yet. I can write a test with Overseas and fix this with that? I don't like making too many changes to the code until I have something to test it with :D

Reply:
Makes sense :)